dbs/mongo/queries/addQuestion.js

```diff
@@ -0,0 +1,28 @@
+const {Questions} = require('../models.js');
+
+async function addQuestion(p_id, name, email, body){
+
+  //let findHighestQ_ID = await Questions.find().sort({question_id: -1}).limit(1);
+
+ // let q_id =  findHighestQ_ID[0].question_id + 1;
+
+  let doc = new Questions({
+    question_id: 1,
```

Comment:
question_id is always 1?

Reply:
nope, that definitely needs to be changed, thank you!